.gitmodules

```diff
@@ -36,7 +36,7 @@
     url = https://github.com/mitsuba-renderer/embree
 [submodule "ext/enoki"]
 	path = ext/enoki
-	url = https://github.com/wjakob/enoki
+	url = git@github.com:wjakob/enoki.git
```

Comment:
Let's undo this, we should definitely clone via `https://` instead of the old-school git protocol. or am I misunderstanding this line?

Reply:
This was back when `wjakob/enoki` was private, and HTTPS cloning is not convenient when 2fac is enabled (`git@github` uses SSH instead).

Reply:
Ah, that makes a ton of sense :-)